runtime-testsuite/test/org/antlr/v4/test/runtime/ExtraTests.java

```diff
@@ -0,0 +1,81 @@
+package org.antlr.v4.test.runtime;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExtraTests {
+	public static RuntimeTestDescriptor[] getRuntimeTestDescriptors(String targetName) {
+		List<RuntimeTestDescriptor> descriptors = new ArrayList<>();
+
+		descriptors.add(getLineSeparatorLfTest(targetName));
+		descriptors.add(getLineSeparatorCrLfTest(targetName));
+		descriptors.add(getLargeLexerDescriptor(targetName));
+
+		return descriptors.toArray(new RuntimeTestDescriptor[0]);
+	}
+
+	static RuntimeTestDescriptor getLineSeparatorLfTest(String targetName) {
+		UniversalRuntimeTestDescriptor result = new UniversalRuntimeTestDescriptor();
+		result.name = "LineSeparatorLf";
+		result.targetName = targetName;
+		result.testType = "Lexer";
+		result.grammar = "lexer grammar L;\n" +
+				"T: ~'\\n'+;\n" +
+				"SEPARATOR: '\\n';";
+		result.grammarName = "L";
+		result.input = "1\n2\n3";
+		result.output = "[@0,0:0='1',<1>,1:0]\n" +
+				"[@1,1:1='\\n',<2>,1:1]\n" +
+				"[@2,2:2='2',<1>,2:0]\n" +
+				"[@3,3:3='\\n',<2>,2:1]\n" +
+				"[@4,4:4='3',<1>,3:0]\n" +
+				"[@5,5:4='<EOF>',<-1>,3:1]\n";
+		return result;
+	}
+
+	static RuntimeTestDescriptor getLineSeparatorCrLfTest(String targetName) {
+		UniversalRuntimeTestDescriptor result = new UniversalRuntimeTestDescriptor();
+		result.name = "LineSeparatorCrLf";
+		result.targetName = targetName;
+		result.testType = "Lexer";
+		result.grammar = "lexer grammar L;\n" +
+				"T: ~'\\r'+;\n" +
+				"SEPARATOR: '\\r\\n';";
+		result.grammarName = "L";
+		result.input = "1\r\n2\r\n3";
+		result.output = "[@0,0:0='1',<1>,1:0]\n" +
+				"[@1,1:2='\\r\\n',<2>,1:1]\n" +
+				"[@2,3:3='2',<1>,2:0]\n" +
+				"[@3,4:5='\\r\\n',<2>,2:1]\n" +
+				"[@4,6:6='3',<1>,3:0]\n" +
+				"[@5,7:6='<EOF>',<-1>,3:1]\n";
+		return result;
+	}
+
+	static RuntimeTestDescriptor getLargeLexerDescriptor(String targetName) {
```

Comment:
Ok, I've moved `LargeLexer` in the current PR for the demo.

Reply:
Ok. Yeah, I'm still thinking we are making a big departure from the point of descriptor files to solve this \r\n thing. Seems we're off the road to shove in a test case for a tiny fraction of the number of tests

Reply:
I have yet another generated test [`getAtnStatesSizeMoreThan65535Descriptor`](https://github.com/antlr/antlr4/pull/3493/files#diff-aa2cc9084de70a064d9035752f274ac7e6551b512db05f5665cc64d92bdd0d2dR84) in another PR. It's too excess to use yet another big file with uniform data.

Reply:
I see. yes, starting to make sense to me for generated examples.